src/apis/follow/index.ts

```diff
@@ -1,6 +1,5 @@
 import axios from 'axios';
 import { API_BASE_URL } from '@constants/Api';
-import { Follow } from '@/types';
 
 const postFollowUser = async (userId: string, token: string) => {
   const response = await axios.post(
```

Comment:
```suggestion
  const response = await axios.post<Follow>(
```
요렇게 하면 반환타입을 지정할 수 있습니다!